src/backend/TrafficCourts/Citizen.Service/Models/Dispute/NoticeOfDispute.cs

```diff
@@ -26,71 +25,68 @@ public class NoticeOfDispute
         /// The date and time the violation ticket was issue. Time must only be hours and minutes.
         /// </summary>
         [JsonPropertyName("issued_date")]
-        public DateTime? IssuedDate { get; set; }
+        public DateTime IssuedDate { get; set; }
 
         /// <summary>
         /// The surname or corporate name.
         /// </summary>
         [JsonPropertyName("surname")]
-        public string? Surname { get; set; }
+        public string Surname { get; set; } = null!;
```

Comment:
As far as I know, this basically says that I'm confident that this property is not going to be null ever since it is expected this property to be assigned with actual value at some point so "!" suppresses the warning and if it somehow becomes null then the validation will throw the error.